nutanix/services/volumesv2/resource_nutanix_volume_group_v2.go

```diff
@@ -361,7 +361,7 @@ func ResourceNutanixVolumeGroupV2Create(ctx context.Context, d *schema.ResourceD
 	uuid := rUUID.EntitiesAffected[0].ExtId
 	d.SetId(*uuid)
 
-	return ResourceNutanixVolumeGroupV2Read(ctx, d, meta)
+	return nil
```

Comment:
Once create is done, we should always read the volume group to get the most recent state of VG